### PR TITLE
Fix/backport mpdf

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

This is a duplicate of #7118 for the hotfix-7.10.x branch.

## Motivation and Context

This allows us to composerify mPDF while still maintaining backwards compatibility by including empty PHP files and aliasing the changed class names.

## How To Test This

1. Check that mPDF works as expected.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: modules/AOS_PDF_Templates/PDF_Lib/classes/meter.php [hunk 1]
```diff
@@ -1,228 +1,4 @@
...
-
-                $svg .= '</g></svg>';
-            }
-        }
-
-        return $svg;
-    }
-}	// end of class
+/**
+ * Class for backwards compatibility only
```
Comment (connorshea) [inline modules/AOS_PDF_Templates/PDF_Lib/classes/meter.php]: We shouldn't bother with it right now, but I wonder if we should print deprecation notices if these files are included? Is there a way to do that, or would Composer autoload these files and cause false positives?

**Comment by connorshea** [comment]
`Warning: Class 'MpdfException' not found in /home/travis/build/salesagility/SuiteCRM/deprecated.php on line 66` 🤔

File: deprecated.php [hunk 2]
```diff
@@ -58,3 +58,14 @@ class_alias('SuiteCRM\\Zend_Oauth_Provider', 'Zend\\Oauth\\Provider');
     /** @deprecated this is an alias for Zend_Oauth_Provider */
     class Provider {}
 }
+
+
+/**
+ * Backwards compatibility for mPDF
+ */
+class_alias('MpdfException', 'mPDF_exception');
```
Comment (connorshea) [inline deprecated.php]: I think you got this backward

Comment (connorshea) [reply deprecated.php]: Or maybe not, why is this erroring 🤔

**Comment by connorshea** [comment]
@Dillon-Brown FWIW, we finally got the backport merged in our 7.10 fork a while ago, and deployed it to production on Monday. It required one change to a PDF Template like I mentioned in my comment on #7118, but other than that we haven't had any problems reported yet.

I'm not sure if we should merge this since it's _technically_ a breaking change, but it also doesn't seem like a good idea to just continue without the fix for all these errors.

**Comment by mattlorimer** [comment]
no longer required

**Pull request closed without merging**